### PR TITLE
Sex form added

### PR DESCRIPTION
**Pull request by lex2001**
YO!

### CODE REVIEW

**Comment by lex2001** [comment]
чо за?

**Comment by CrazyFlasher** [comment]
please, remove all redundant files added

**Comment by CrazyFlasher** [comment]
please, fix code format in changed as files

File: examples/simpleMVC/src/com/domwires/example/simplemvc/context/AppContext.as [hunk 1]
```diff
@@ -55,6 +56,8 @@ package com.domwires.example.simplemvc.context
 			map(AppViewMessage.FIRST_NAME_CLICKED, GenerateFirstNameCommand);
 			map(AppViewMessage.LAST_NAME_CLICKED, GenerateLastNameCommand);
 			map(AppViewMessage.AGE_CLICKED, GenerateAgeCommand);
+            map(AppViewMessage.SEX_CLICKED, GenerateSexCommand);
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/context/AppContext.as:59]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as [hunk 2]
```diff
@@ -10,6 +10,16 @@ package com.domwires.example.simplemvc.model
 		private var _firstName:String;
 		private var _lastName:String;
 		private var _age:int;
+		private var _sex:String;
+
+        public function setSex(value:String):IAppModel
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as:15]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as [hunk 3]
```diff
@@ -52,5 +62,10 @@ package com.domwires.example.simplemvc.model
 		{
 			return _age;
 		}
+
+        public function get sex():String
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as:66]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModelMessage.as [hunk 4]
```diff
@@ -10,6 +10,7 @@ package com.domwires.example.simplemvc.model
 		public static const FIRST_NAME_CHANGED:AppModelMessage = new AppModelMessage("FIRST_NAME_CHANGED");
 		public static const LAST_NAME_CHANGED:AppModelMessage = new AppModelMessage("LAST_NAME_CHANGED");
 		public static const AGE_CHANGED:AppModelMessage = new AppModelMessage("AGE_CHANGED");
+        public static const SEX_CHANGED:AppModelMessage = new AppModelMessage("SEX_CHANGED");
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModelMessage.as:13]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/IAppModel.as [hunk 5]
```diff
@@ -11,5 +11,6 @@ package com.domwires.example.simplemvc.model
 		function setFirstName(value:String):IAppModel;
 		function setLastName(value:String):IAppModel;
 		function setAge(value:int):IAppModel;
+        function setSex(value:String):IAppModel;
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/IAppModel.as:14]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/IAppModelImmutable.as [hunk 6]
```diff
@@ -10,5 +10,6 @@ package com.domwires.example.simplemvc.model
 		function get firstName():String;
 		function get lastName():String;
 		function get age():int;
+        function get sex():String;
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/IAppModelImmutable.as:13]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 7]
```diff
@@ -29,6 +29,7 @@ package com.domwires.example.simplemvc.view
 		private var firstNameLabel:Label;
 		private var lastNameLabel:Label;
 		private var ageLabel:Label;
+        private var sexLabel:Label;
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:32]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 8]
```diff
@@ -42,14 +43,21 @@ package com.domwires.example.simplemvc.view
...
 			lastNameLabel = createForm("Last name:", layoutGroup, onLastNameClicked);
 			ageLabel = createForm("Age:", layoutGroup, onAgeClicked);
+			sexLabel = createForm("Sex:", layoutGroup, onSexClicked);
 
 			viewContainer.addChild(layoutGroup);
 
 			addMessageListener(AppModelMessage.FIRST_NAME_CHANGED, firstNameChanged);
 			addMessageListener(AppModelMessage.LAST_NAME_CHANGED, lastNameChanged);
 			addMessageListener(AppModelMessage.AGE_CHANGED, ageNameChanged);
+            addMessageListener(AppModelMessage.SEX_CHANGED, sexNameChanged);
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:53]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 9]
```diff
@@ -42,14 +43,21 @@ package com.domwires.example.simplemvc.view
...
 		}
 
+        private function sexNameChanged(m:IMessage):void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:56]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 10]
```diff
@@ -106,5 +114,10 @@ package com.domwires.example.simplemvc.view
 		{
 			dispatchMessage(AppViewMessage.AGE_CLICKED, null, true);
 		}
+
+        private function onSexClicked():void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:118]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppViewMessage.as [hunk 11]
```diff
@@ -10,6 +10,7 @@ package com.domwires.example.simplemvc.view
 		public static const FIRST_NAME_CLICKED:AppViewMessage = new AppViewMessage("FIRST_NAME_CLICKED");
 		public static const LAST_NAME_CLICKED:AppViewMessage = new AppViewMessage("LAST_NAME_CLICKED");
 		public static const AGE_CLICKED:AppViewMessage = new AppViewMessage("AGE_CLICKED");
+        public static const SEX_CLICKED:AppViewMessage = new AppViewMessage("SEX_CLICKED");
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppViewMessage.as:13]: improve code format

**Review by CrazyFlasher — CHANGES_REQUESTED** [review]
Attach screenshot of final build

**Pull request closed without merging**